gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/exclusion/Exclusion.java

```diff
@@ -49,9 +49,9 @@
 public class Exclusion implements BinaryGraphToGraphOperator {
 
   /**
-   * Creates a new logical graph containing only vertices and edges that exist
-   * in the first input graph but not in the second input graph. Vertex and edge
-   * equality is based on their respective identifiers.
+   * Reduces the first input graph to contain only vertices and edges that don't exist in
```

Comment:
Good description. Please move it from the execute method to the class javadoc. See #1240